lib-multisrc/grouple/src/eu/kanade/tachiyomi/multisrc/grouple/GroupLe.kt

```diff
@@ -213,15 +212,9 @@ abstract class GroupLe(
 
     private fun chapterListParse(response: Response, manga: SManga): List<SChapter> {
         val document = response.asJsoup()
-        if ((
-            document.select(".expandable.hide-dn").isNotEmpty() && document.select(".user-avatar")
-                .isEmpty() && document.toString()
-                .contains("current_user_country_code = 'RU'")
-            ) || (
-                document.select("img.logo")
-                    .first()?.attr("title")
-                    ?.contains("Allhentai") == true && document.select(".user-avatar").isEmpty()
-                )
+
+        if (document.select(".user-avatar").isEmpty() &&
+            document.title().run { contains("AllHentai") || contains("MintManga") || contains("МинтМанга") }
```

Comment:
any way to do it without depending on title? and why is this check repeated here and at line 310?

Reply:
I can tweak the checks to use the syntax from the body if it’s needed. About the repeated checks, I just fixed the broken ones that were already there, and I didn’t think they were too much.

Reply:
I think those sites can be accessed without authorization. Only difference will be that chapter images will be replaced with placeholders with "Login to see image" text. Like this: https://2.mintmanga.one/dnevnik_plohih_myslei__A35c96/vol1/60?d=MjE1Nzky&mtr=true
No extra error messages or redirections to login screen to do the check without depending on a title.

I think code at line 310 used to download images of a single chapter, while this line are used by function at 197 line
`override fun fetchChapterList`
They might do different things, download images/getting list of chapters from site.

Reply:
yea misunderstood the code/intent from the pr diff